api/configuration_services.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/lyft/protoc-gen-validate/validate/validate.proto";
+import "google/api/annotations.proto";
+
+package ttn.lorawan.v3;
+
+option go_package = "go.thethings.network/lorawan-stack/pkg/ttnpb";
+
+message ListFrequencyPlansRequest {
+  // Optional base frequency in MHz for hardware support (433, 470, 868 or 915)
+  uint32 base_frequency = 1;
```

Comment:
The comment implies that there are only 4 possible values, why is this field not validated?

Reply:
These are examples and are subject to change. No reason for API validation here.

Reply:
The comment should mention `e.g` or `for example` then.